pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -124,6 +126,9 @@ func (ds *dockerService) PodSandboxStatus(podSandboxID string) (*runtimeApi.PodS
 	}
 
 	labels, annotations := extractLabels(r.Config.Labels)
+	// Delete the container name label for infra container. It is added in dockershim, should
+	// not be exposed to CRI.
+	delete(labels, types.KubernetesContainerNameLabel)
```

Comment:
The only problem is that, we should only delete the label for pod sandbox.
If we want to move this inside `extractLabels`, we may need to add some special logic to and and remove labels for sandbox, or add a separate `makeSandboxLabels` and `extractSandboxLabels`.